glibc/README.md

```diff
@@ -0,0 +1,57 @@
+# Glibc base image
+
+This image compiles glibc for use with Alpine. 
+
+In order to build and push this image, you must build and push an image for each target architecture
+(`amd64` and `arm64`). Seeing as this is a compilation task, you must do this on a machine that uses
+the target architecture, push the images to Github, then create a consoildated manifest. This readme
+will guide you through how to do this.
+
+## Log into the Github Docker Registry
+
+First generate a Github personal access token with write permission to the package registry (you will
+also have to grant the token SSO access for the `permutive-engineering` organisation.
+
+Once you have you token use `docker login` to gain access to the registry. Use your Github username
+as the username and the token you have just generated as the password:
+
+```bash
+docker login ghcr.io/permutive-engineering
+```
+
+## Build and Push the Architecture Specific Image
+
+Use the command below while in this directory to build and push the image, substituting `<amd64|arm64>`
+for your current architecture.
+
+```bash
+docker build . -t ghcr.io/permutive-engineering/permutive-glibc:<amd64|arm64>
+docker push ghcr.io/permutive-engineering/permutive-glibc:<amd64|arm64>
+```
+
+> ⚠️ Make sure you have pushed images for both `amd64` and `arm64` architectures before proceeding to
+> the next step.
+
+## Create a Multi-Architecture Manifest
+
+This final step will create a manifest that includes both images.
+
+First, create a new `buildx` instance if it doesn't already exist on your system:
+
+```bash
+docker buildx create --name glibc
+docker buildx use glibc
+docker buildx inspect --bootstrap
```

Comment:
**thought:** Feel free to ignore, but I think you can simplify this a bit.

I think you can do `create --use --name glibc` to switch immediately. I also think if you try to use the builder without it being bootstrapped it will do that for you. So the `docker buildx imagetools...` call below will bootstrap there.

<sub>This comment follows the [**conventionalcomments.org**](https://conventionalcomments.org/) standard</sub>

Reply:
> I think you can do create --use --name glibc to switch immediately

Yep, the problem is if someone already has the `glibc` instance locally, it won't switch. This allows the first command to fail.

>  I also think if you try to use the builder without it being bootstrapped

I wanted to be careful here again, the `--bootstrap` ensures it definitely is set up before actually building anything.